tests/Integration/RequestIntegrationTest.php

```diff
@@ -74,9 +75,9 @@ public function applyToEventWithRequestHavingIpAddressDataProvider(): array
     public function testApplyToEvent(array $options, ServerRequestInterface $request, array $expectedResult): void
     {
         $event = new Event();
-        $integration = new RequestIntegration(new Options($options));
+        $integration = new RequestIntegration(new Options($options), $this->createRequestFactory($request));
 
-        RequestIntegration::applyToEvent($integration, $event, $request);
+        RequestIntegration::applyToEvent($integration, $event);
```

Comment:
This change is causing an unwanted code coverage drop as we're not testing anymore that the `applyToEvent` method works when a custom request is passed to it

Reply:
So what's your suggestion here? Copy the test and refactor that one to match `IgnoreErrorsIntegrationTest`?

Reply:
I don't really expect this method to change in any way, it should just pass as it did before. The method that should be refactored is `testInvokeWithRequestHavingIpAddress`, and to clarify what I meant is that it should be changed to match what have been done in the `IgnoreErrorsIntegrationTest` class